### PR TITLE
CDC documentation

### PR DESCRIPTION
**Pull request by l-heemann**
Documents CDC in cypher manual.

**Note:** Adds links to unpublished [docs-cdc](https://github.com/neo4j/docs-cdc).

### CODE REVIEW

**Comment by l-heemann** [comment]
[(Publish when public beta, Nodes 2023) CDC Documentation](https://trello.com/c/XoRQNjlu/3224-publish-when-public-beta-nodes-2023-cdc-documentation)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by HannesSandberg — APPROVED** [review]
Looks good to me 👍 Can not verify the link but trusting you that it is pointing to the right place

**Comment by l-heemann** [comment]
> Looks good to me 👍 Can not verify the link but trusting you that it is pointing to the right place

I am hoping for the docs reviewer to verify it.

File: modules/ROOT/pages/clauses/listing-procedures.adoc [hunk 1]
```diff
@@ -152,6 +152,21 @@ SHOW PROCEDURES
 |===
 | name | description | mode | worksOnSystem
 
+| "cdc.current"
+| "Returns the current change identifier that can be used to stream changes from."
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/listing-procedures.adoc:156]:
```suggestion
| "Returns the current change identifier that can be used to stream changes from."
label:new[Introduced in 5.13]
```

File: modules/ROOT/pages/clauses/listing-procedures.adoc [hunk 2]
```diff
@@ -152,6 +152,21 @@ SHOW PROCEDURES
...
+| "READ"
+| false
+
+| "cdc.earliest"
+| "Returns the earliest change identifier that can be used to stream changes from."
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/listing-procedures.adoc:161]:
```suggestion
| "Returns the earliest change identifier that can be used to stream changes from."
label:new[Introduced in 5.13]
```

File: modules/ROOT/pages/clauses/listing-procedures.adoc [hunk 3]
```diff
@@ -152,6 +152,21 @@ SHOW PROCEDURES
...
+| "READ"
+| false
+
+| "cdc.query"
+| "Query changes happened from the provided change identifier."
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/listing-procedures.adoc]: I don't understand what "happened from the provided change identifier" means (hence my suggestion may be inaccurate).

```suggestion
| "Query changes observed by the provided change identifier."
label:new[Introduced in 5.13]
```

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 4]
```diff
@@ -57,6 +57,20 @@ For all available Cypher types, see the section on xref::values-and-types/proper
...
+label:beta[]
+
+[source, cypher, role=noheader]
+----
+CALL cdc.current()
+CALL cdc.earliest()
+CALL cdc.query(from, selectors)
+----
+
+| Introduction of the change data capture feature. For details see link:{neo4j-docs-base-uri}/cdc/{page-version}/introduction[Change Data Capture]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: It is always capitalised when Ive read about it in internal release notes and elsewhere.

```suggestion
| Introduction of the Change Data Capture (CDC) feature.
For details see link:{neo4j-docs-base-uri}/cdc/{page-version}/introduction[Change Data Capture]
```

**Review by JPryce-Aklundh — COMMENTED** [review]
A few suggestions, @l-heemann
With regard to the link to the unpublished docs, myself and @recrwplay (on holiday today) will have a look and see if we can get the CDC docs out in time of 5.13. If yes, then no problem.
If no, we may have to find a temporary location to link to.

**Comment by l-heemann** [comment]
@JPryce-Aklundh So should we merge this now or do you want to merge it later?

**Comment by JPryce-Aklundh** [comment]
@l-heemann let's wait with merging until we have figured out the situation with the CDC docs.
This remainder of this PR (i.e. everything apart from the link) is fine now, though.

**Comment by ali-ince** [comment]
@JPryce-Aklundh I think we now have a plan for cdc docs too: https://neo4j.slack.com/archives/C02JR5E4L/p1695305000000069. Is it ok to merge this now, wdyt?

**Comment by JPryce-Aklundh** [comment]
@ali-ince - yes, that is fine. I just noticed that the labels that I added as a suggestion does not render well (and are not needed either as they are in the Ops Manual), so I will fix it and then merge.

File: modules/ROOT/pages/clauses/listing-procedures.adoc [hunk 5]
```diff
@@ -152,6 +152,24 @@ SHOW PROCEDURES
...
+| "Returns the earliest change identifier that can be used to stream changes from."
+label:new[Introduced in 5.13]
+| "READ"
+| false
+
+| "cdc.query"
+| "Query changes observed by the provided change identifier."
+label:new[Introduced in 5.13]
+| "READ"
+| false
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/listing-procedures.adoc]:
```suggestion
| "cdc.current"
| "Returns the current change identifier that can be used to stream changes from."
| "READ"
| false

| "cdc.earliest"
| "Returns the earliest change identifier that can be used to stream changes from."
| "READ"
| false

| "cdc.query"
| "Query changes observed by the provided change identifier."
| "READ"
| false
```

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 6]
```diff
@@ -57,6 +57,21 @@ For all available Cypher types, see the section on xref::values-and-types/proper
...
+
+[source, cypher, role=noheader]
+----
+CALL cdc.current()
+CALL cdc.earliest()
+CALL cdc.query(from, selectors)
+----
+
+| Introduction of the Change Data Capture (CDC) feature.
+For details see link:{neo4j-docs-base-uri}/cdc/{page-version}/introduction[Change Data Capture]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
For details, see link:{neo4j-docs-base-uri}/cdc/{page-version}/introduction[Change Data Capture].
```

**Pull request merged**